pints/_transformation.py

```diff
@@ -0,0 +1,574 @@
+#
+# Transformation functions
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+from scipy.special import logit, expit
+
+
+class Transform(object):
+    """
+    Abstract base class for objects that provide some convenience parameter
+    transformation from the model parameter space to a search space.
+
+    If ``t`` is an instance of a ``Transform`` class, you can apply
+    the transformation of a parameter vector from the model space ``p`` to the
+    search space ``x`` by using ``x = t.to_search(p)`` and the inverse by using
+    ``p = t.to_model(x)``.
+    """
+    def apply_log_pdf(self, log_pdf):
```

Comment:
I'm not sure "apply_" is the right term, so maybe replace with just "def log_pdf(..)" ?

Reply:
Or "convert_log_pdf" etc?

Reply:
Cool, either "convert_log_pdf" or just "log_pdf" sounds good. So which one? 😄

Reply:
Maybe "wrap_" :thinking:

Reply:
Oh no this one's not wrapped :D

How about "convert_x" for all these methods?

Reply:
Or "apply_to_", so it becomes logtransform.apply_to_log_pdf()". But I still favour convert_

Reply:
Changed to "convert_".